recipes/metalign/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.10.1" %}
+
+package:
+  name: metalign
+  version: {{ version }}
+
+source:
+  url: https://github.com/nlapier2/Metalign/archive/v{{ version }}.tar.gz
+  sha256: 15ff4ce2aeb015fbfc6b01c8baf4e1fa75ed43c45467c795dd7df364899cce16
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
@nlapier2 This is probably the line causing the Circle CI error. You’ll need to either:
1. Create a `setup.py` in Metalign or
2. Find an alternate install pip install command

I assume 1 would be easier due to Metalign not having any dependencies besides KMC, minimap2 and CMash, so the setup.py would be straightforward. I.e. Create a minimal setup.py as if you intend to release on PyPi